bash_completion

```diff
@@ -92,5 +92,5 @@ if [[ -n ${ZSH_VERSION-} ]]; then
   autoload -U +X bashcompinit && bashcompinit
 fi
 
-complete -o default -o nospace -F __nvm nvm
+complete -o default -F __nvm nvm
```

Comment:
If you type `nvm a[tab` it'll give you only `nvm alias`, so if you want to keep entering the next parameter, you'll need to manually add another space here, then you can press [tab] again, or it'll not work. By removing `-o no space`, the same situation, it'll give you `nvm alias `, and you can directly press the next `[tab]` for the next parameter, that's what most of the bash_completions doing, I don't know why we should disable it here.